### PR TITLE
Fix Google Drive File Stream package filename

### PR DESCRIPTION
**Pull request by hangrymuppet**
Google has changed the filename of the installer package.

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [X] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [X] `brew audit --cask {{cask_file}}` is error-free.
- [X] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [ ] `brew install --cask {{cask_file}}` worked successfully.
- [ ] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Thank you @hangrymuppet.

**Review by miccal — DISMISSED** [review]

**Comment by miccal** [comment]
@vitorgalvao thoughts on the token reference here?

File: Casks/google-drive-file-stream.rb [hunk 1]
```diff
@@ -8,12 +8,13 @@
 
   depends_on macos: ">= :el_capitan"
 
-  pkg "GoogleDriveFileStream.pkg"
+  pkg "GoogleDrive.pkg"
 
   uninstall login_item: "Google Drive File Stream",
```
Comment (vitorgalvao) [inline Casks/google-drive-file-stream.rb:13]: Is this still correct?

**Comment by vitorgalvao** [comment]
In PKGs, what matters is what’s inside. In this case it’s `Google Drive.app`, among others; I’m not convinced we should rename this one.

**Comment by sparhomenko** [comment]
If that helps: according to [Google documentation](https://support.google.com/googleone/answer/10309431):
> Drive File Stream will now be called Google Drive for desktop
As part of that the alternative software (Backup and Sync) is planned to be deprecated.

**Comment by vitorgalvao** [comment]
> If that helps

It does! The cask should be renamed to `google-drive.rb`. Yet, this is popular so we should do a transition like [from `zoomus` to `zoom`](https://github.com/Homebrew/homebrew-cask/blob/db53068983dd81cce87354e2d52cfabc689a340e/Casks/zoomus.rb#L11L27).

[Maybe we can do better](https://github.com/Homebrew/homebrew-cask/pull/99419). Apologies for closing your PR, @hangrymuppet, but this may end up being a complex change that will require a lot of maintainer back and forth, so it’s easier if we do it ourselves.

Thank you for the contribution.

**Pull request closed without merging**